doc/source/conf.py

```diff
@@ -301,17 +301,29 @@ def intersphinx_pyansys_geometry(switcher_version: str):
 latex_elements = {"preamble": latex.generate_preamble(html_title)}
 sd_fontawesome_latex = True
 
-linkcheck_exclude_documents = ["index", "getting_started/local/index", "assets"]
+linkcheck_exclude_documents = ["index", "getting_started/local/index"]
 linkcheck_ignore = [
-    r"https://github.com/ansys/pyansys-geometry-binaries/.*",
+    r"https://github.com/ansys/pyansys-geometry-binaries",
     r"https://download.ansys.com/",
     r".*/examples/.*.py",
     r".*/examples/.*.ipynb",
+    r"_static/assets/.*",
 ]
 
+# If we are on a release, we have to ignore the "release" URLs, since it is not
```

Comment:
Should "URLs" be plural because "release" is singular?

Reply:
There are several URLs related to "release". Therefore the plurar. I believe its fine :)